ssg/templates.py

```diff
@@ -3,8 +3,7 @@
 
 import os
 import sys
-import re
-from xml.sax.saxutils import unescape
+import imp
```

Comment:
The docs say that it's deprecated: https://docs.python.org/3/library/imp.html

Reply:
Well, that's the problem of being compatible with older versions of Python.So far it works.

Reply:
what if we detect which version of python is being executed and have a separate code for each version. I know that it sounds cumbersome, but when we stop using python2 we can just get rid of the older implementation.